src/test/resources/test_setup.yaml

```diff
@@ -56,6 +56,7 @@ config:
         prefix: 'local' # The prefix to match on incoming IDs
         origin: 'ds.test'
         description: 'Local records, mostly biographical'
+        getrecordendpoint: 'http://example.com/ds-present/v1/record/'
         # Temporary hack to make the Preservica XML block well-formed (it is missing a namespace declaration)
         testdatahack: &TEST_DATA_HACK_XIP
           regexp: '<xip:(DeliverableUnit|Collection|Manifestation) +status="([^"]+)" *>'
```

Comment:
Is this still needed after the changes to `ds-datahandler`?

Reply:
Nope, but you are to far ahead. KB internal issue DISC-142 is about removing this. I can remove it it in this PR if you like?

Reply:
Sorry, I did not see that it would be handled elsewhere. My bad. But fine to remove it here anyway.